apps/prairielearn/src/question-servers/calculation-worker.js

```diff
@@ -68,7 +78,16 @@ function generate(server, coursePath, question, variant_seed) {
   };
 }
 
+/**
+ * @param {any} server
+ * @param {string} coursePath
+ * @param {import('../lib/db-types.js').Submission} submission
+ * @param {import('../lib/db-types.js').Variant} variant
+ * @param {import('../lib/db-types.js').Question} question
+ */
```

Comment:
I deliberately avoided adding a return type for now. That'll require a change to the `GradeResultData` type, but I want to make that change after it's moved to a new file so that the change is visible in the diff and isn't obscured by the move.